src/js/components/form-elements/Button.jsx

```diff
@@ -5,32 +5,29 @@ import PropTypes from 'prop-types';
 import Translate from 'utils/Translate';
 
 const Button = ({
-  label, defaultLabel, disabled, variant, type, onClickAction, EndIcon,
-}) => (
-  <button
-    className={`${variant}-button d-flex justify-content-around align-items-center ${disabled && `${variant}-button-disabled`} ${variant === 'dropdown' && 'dropdown-toggle'}`}
-    disabled={disabled}
-    type={type}
-    onClick={onClickAction}
-    data-toggle={variant === 'dropdown' && 'dropdown'}
-    aria-haspopup={variant === 'dropdown' && 'true'}
-    aria-expanded={variant === 'dropdown' && 'false'}
-  >
-    {EndIcon ? (
+  label, defaultLabel, disabled, variant, type, onClick, EndIcon, isDropdown,
```

Comment:
Instead of having dropdown set by `variant` prop I added `isDropdown` prop for that and `variant` is responsible for setting button styling eg: **primary**, **transparent**, **secondary**